po/de.po

```diff
@@ -2746,17 +2750,20 @@ msgstr "Verbesserung für Flak"
 msgid "High Explosive Flak shards"
 msgstr "Hochexplosive Flak-Splittermunition"
 
+# Basierend auf Deinem Kommentar, dass Flak wortgetreu entsprechend der
+# Abkürzung übersetzt werden soll habe ich die Flugabwehrgeschütze in
+# Flugabwehrkanonen umbenannt -alfred007
```

Comment:
Das ist gut, müsste man streng genommen aber bei sämtlichen Waffen machen, die im Spiel "Cannon" heißen und mit "Geschütz" übersetzt werden. Waffentechnisch ist "[Geschütz](https://de.wikipedia.org/w/index.php?title=Gesch%C3%BCtz&oldid=186112454)" ein Oberbegriff für Haubitzen/Mörser und Kanonen.

Denkst du, dass wir das umsetzen sollten?

Reply:
Wenn wir schon dabei sind, sprachliche Feinheiten umzusetzen, sollten wir das auch hier tun.

Reply:
Ich denke, dass die Änderung von "Geschütz" in "Kanone" in einen neuen Commit basierend auf einer aktuelleren de.po fließen sollte. Bist du damit einverstanden?

Reply:
Können wir so machen. Ich übernehme gerne die Arbeit.

Reply:
Gut, dann aber erst mal das hier abschließen :) Ich lasse das hier offen, damit es beim Durchscrollen leichter auffällt.